music_assistant/common/models/player_queue.py

```diff
@@ -36,6 +37,7 @@ class PlayerQueue(DataClassDictMixin):
     current_item: QueueItem | None = None
     next_item: QueueItem | None = None
     radio_source: list[MediaItemType] = field(default_factory=list)
+    played_media_item: list[MediaItemType] = field(default_factory=list)
```

Comment:
hmmm, not sure about this variable name as its a bit misleading Can you at least add some surrounding comments what its for.

Reply:
Agree. How about `enqueued_media_items` ?

Reply:
would it maybe be possible to limit this to only the uri's (or Itemmapping) of the items you want to remember ? 
Otherwise the player_queue object which is already very large, gets even larger. Note that this also has to be serialized over the api

Reply:
I am currently limiting this to 10 MediaItems using a FIFO approach here: https://github.com/music-assistant/server/blob/10fb892d31948f1955881d52b985c26126a47a4d/music_assistant/server/controllers/player_queues.py#L369 . Is this what you meant?

Reply:
No, What I mean is instead of storing complete MediaItem objects in that list, will it suffice to only store uri's (so a list of string) ?

Reply:
Yea good one, I think we can store the uri's and translate the uri's into MediaItem object at the time when DSTM kicks in?

Reply:
yes, that way you can also store more items in the list and it will prevent the queueitem object blowing up too much

Reply:
Fixed!